pkg/controller/directvolumemigration/pvcs.go

```diff
@@ -49,7 +51,7 @@ func (t *Task) createDestinationPVCs() error {
 
 		// Adjusting destination PVC storage size request
 		// max(requested capacity on source, capacity reported in migplan, proposed capacity in migplan)
-		if matchingMigPlanPV != nil {
+		if matchingMigPlanPV != nil && settings.Settings.DvmOpts.EnablePVResizing {
```

Comment:
Why do we have a switch for enabling PV resizing specifically for DVM? Maybe there's a good reason, I was kind of expecting a global on/off.

Reply:
We decided to decouple the analysis and actual resizing feature. @pranavgaikwad feel free to add more info or correct this.

Reply:
@pranavgaikwad so there is a separate setting on the plan? Or is this the only option

Reply:
@djwhatle see my comment here: https://github.com/konveyor/mig-controller/pull/965#discussion_r580533976